docs/source/using/addons.adoc

```diff
@@ -307,6 +307,22 @@ $ minishift addons uninstall admin-user
 Add-on 'admin-user' uninstalled
 ----
 
+[[available-addons]]
+== Available Add-ons
+
+The add-ons link:github.com/minishift/minishift-addons[repository] currently has the following add-ons:
+
+- link:https://github.com/minishift/minishift-addons/tree/master/add-ons/che[Eclipse Che]
```

Comment:
I am not a big fan of listing add-ons like this, especially with no additional information what the add-on does.
This is so easy to get out of sync. I rather have a generic section about the minishift-addons and how to use it with a link into the add-ons directory. If we curate a manual list, I think some more context would be useful, eg in form of a table.

Reply:
-1 For listing add-ons from minishift-addons repository in Minishift documentation as it will be a lot of maintenance to keep this always up to date.  AFAIK the issue https://github.com/minishift/minishift/issues/1389 was about adding information about the default add-ons. @budhrg Can you confirm?

Reply:
+1 for expanding on the default add-ons and as mentioned inhttps://github.com/minishift/minishift/pull/1640#discussion_r147941078, I am all up for writing a section about the minishift-addon repo. However, in a generic way, w/o listing manually existing add-ons.

Reply:
If we could fetch the list of addons in the repo automatically using some github hooks that'd be great, IMO

Reply:
That would help :+1: